<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmbeddedSysEmu</class>
 <widget class="QMainWindow" name="EmbeddedSysEmu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmbeddedSysEmu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="telemetryDisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="telemetryLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>199</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Current Telemetry</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFrequency">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output frequency (milliseconds)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setFrequency">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set frequency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setIPaddress">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>IP Address of SCE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipaddress">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
